### DISCUSSION TITLE
How cloudwash mark resources those needs to be cleaned?

### DISCUSSION THREAD

**Discussion by lyfofvipin** [RedHatQE/cloudwash#35] [Q&A] [answered]
Hi all,
I am bit curious about how cloudwash mark resources those needs to be cleaned?
did it check the load on given resources or it's something else?

**Comment by lyfofvipin** [comment, marked as answer]
Okay got the answer it's based on the template, https://github.com/RedHatQE/cloudwash/blob/master/settings.yaml.template
We can set things.

**Comment by jyejare** [comment]
Right! 

The marking varies based on resource type to type.

E.g:
- VMs deletion has 2 markings (selection criteria) based on prepended text to the VM hostname and the second criteria is based on the VM running time in minutes.
- Discs, NiCs, and Public IPs when unassigned (unutilized) to any VM they are eligible for cleanup.
- Other resources also follows some/same sort of selection criteria

I could modify https://github.com/RedHatQE/cloudwash/blob/master/settings.yaml.template to clear things more precisely!

Opened issue https://github.com/RedHatQE/cloudwash/issues/50

**Comment by jyejare** [comment]
The Setting file is now modified to be more precise in the 1.0.1 release ! Please take a look at :

https://github.com/RedHatQE/cloudwash/blob/master/settings.yaml.template
